### PR TITLE
Add support for Rider 2018.1.3

### LINKED ISSUE #19: Rider support
Hey @aelij,

thanks for this great extension. I'm wondering if we could start adding support for Rider as well... I can even help with that if you're interested.

Please let me know what you think.
Matt

### PR DESCRIPTION
**Pull request by Socolin**
Based on this documentation https://github.com/JetBrains/resharper-devguide/blob/2b5394f5250709899599cb34cb0dba5fdb776b88/Products/Rider.md

With the help of @matkoch

The build generate a plugin directory, it juste need to be zipped (like with 7-zip or `zip` command on linux)

Fixes #19

### CODE REVIEW

**Comment by aelij** [comment]
Thanks for the contribution! So is this ready to go? Have you tested it in Rider?

**Comment by Socolin** [comment]
Hi, I'm using it with rider currently and it works great.

After compile you can just use 7zip to create a zip of directory plugin and then you can publish it here https://plugins.jetbrains.com/plugin/add/intellij

File: src/Arbel.ReSharper.ConfigureAwaitPlugin/META-INF/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<idea-plugin>
+    <depends>com.intellij.modules.rider</depends>
+    <idea-version since-build="181.4952.311" until-build="183.*"/>
```
Comment (matkoch) [inline src/Arbel.ReSharper.ConfigureAwaitPlugin/META-INF/plugin.xml:4]: This should be `182.*`

Comment (aelij) [reply src/Arbel.ReSharper.ConfigureAwaitPlugin/META-INF/plugin.xml:4]: I've already uploaded the plugin. What would happen if it remained like this? Can it wait for the next version of the plugin (probably next wave of R#)?

Comment (matkoch) [reply src/Arbel.ReSharper.ConfigureAwaitPlugin/META-INF/plugin.xml:4]: Nothing bad will happen, but as soon as 2018.2 is released, this plugin will be shown as compatible, although it won't be. Maybe just release asap :)

**Comment by aelij** [comment]
Do you know what's required to update the plugin to 2018.2? I couldn't find a place where versioning was clearly specified.

**Comment by matkoch** [comment]
since-build > 182.4231.193
until-build > 182.*

**Pull request merged**